config/config.go

```diff
@@ -141,24 +148,33 @@ func (c BlockManagerConfig) Validate() error {
 		}
 	}
 
-	if c.BatchSubmitMaxTime <= 0 {
+	if c.BatchSubmitTime <= 0 {
```

Comment:
In general would make the comparison which check the same variable if possible in one line and just tell the range it should be within vs doing each comparison with it's own error message.